web/packages/teleterm/src/env.ts

```diff
@@ -0,0 +1,24 @@
+interface Env {
+  prehogAddress: string;
+  feedbackAddress: string;
+}
+
+let ENV: Env;
```

Comment:
The name `ENV` implies that the values come from env variables, similar to `process.env` but this is not the case here.

What about something like `staticConfig`?

Reply:
Good idea, done